tv/CMakeLists.txt

```diff
@@ -7,9 +7,6 @@ endif()
 
 if (CLANG_PLUGIN)
   message(STATUS "Clang plugin support enabled")
-  set_target_properties(tv PROPERTIES
-      COMPILE_FLAGS "-DCLANG_PLUGIN"
-  )
```

Comment:
I removed this -DCLANG_PLUGIN because the code block in `tv.cpp` surrounded by CLANG_PLUGIN macro was needed to support NPM.

Reply:
ok, can you remove references to it from the README as well pls?

Reply:
Uhm, is it worth it to keep the cmake flag just to skip creation of the two scripts? Probably not; I would remove it as well